app/services/converter/base.rb

```diff
@@ -1,7 +1,7 @@
 module Converter
   class Base
-    def build_mutation(name, data)
-      data = cleanup(data) unless name.downcase.include?("update")
+    def build_mutation(name, data, update = false)
+      data = cleanup(data) unless name.downcase.include?("update") || update
```

Comment:
das funktioniert technisch weil hier die Definition "update" in den Params der Methode Vorrang hat, aber beim ersten Lesen habe ich gedacht, es ist die oben definierte Methode "update" gemeint gewesen.

Reply:
und ich bin leider zu doof für diese doppelte Verneinung mit "Oder".